EIPS/payoutable-token-draft.md

```diff
@@ -0,0 +1,302 @@
+---
+eip: <to be assigned>
+title: Payoutable Token
+author: Fernando Paris <fer@io.builders>
+status: Draft
+type: Standards Track
+category:  ERC
+created: 2019-05-10
+requires: 20
+
+---
+
+## Simple Summary
+An extension to the ERC-20 standard token that allows Token wallet owners to request payut from their wallet, by calling the smart contract and attaching a payout instruction string.
+
+## Actors
+
+#### Token Wallet Owners
+The person or company who owns the wallet, and will order payout.
+
+#### Token contract operator
+The entity, company responsible/owner of the token contract, and token issuing/minting. This actor is in charge of trying to fullfill all payout request(s), reading the payout instruction(s), and corelate the payout details.
+
+#### Orderer
+An actor who is enable to initiate payout orders on behalf ot a token wallet owner.
+
+
+## Abstract
+Token wallet owners (or approved addresses) can order payout requests through  blockchain. This is done by calling the ```orderPayoutFrom``` or ```orderPayoutFrom``` methods, which initiate the workflow for the token contract operator to either honor or reject the payout request. In this case, payout instructions are provided when submitting the request, which are used by the operator to determine the destination of the funds.
+
+In general, it is not advisable to place explicit routing instructions for the payouts on a verbatim basis on the blockchain, and it is advised to use a private communication alternatives. such as private channels, encrypted storage or similar,  to do so (external to the blockchain ledger). Another (less desirable) possibility is to place these instructions on the instructions field on encrypted form.
+
+
+## Motivation
+Nowadays most of the token payout requests,  a previous centralized transaction, to be able to define the payout destination to be able to execute the payout (burn transaction).
+In the aim of trying step by step to bring all the needed steps into decentralization, exposing all the needed steps of token lifecycle and payment transactions, a payout request can allow wallet owner to initiate the payout order via blockchain.
+Key benefits:
+
+* Payout, burning  traceability is enhanced bringing the initation into the ledger. All payment, payout statuses can be stored on chain.
+* Almost all money/token lifecycle is covered via an decentralized approach, complemented with private communications which is common used in the ecosystem.
+
+
+
+In this case, the following movement of tokens are done as the process progresses:
+
+* Upon launch of the payout request, the appropriate amount of funds are placed on a hold with no notary (i.e. it is an internal hold that cannot be released), and the payout is placed into a ```Ordered``` state
```

Comment:
As far as I understand it, there is always a notary on the hold, but during `orderPayout` the notary is not defined by the user, but set internally to a predefined value (e.g. the owner of the contract)